src/components/AuthLogo/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import logoWhite from 'assets/images/logo-white.png'
+
+type AuthLogoProps = {
+  label: string
+}
+
+const AuthLogo = ({ label }: AuthLogoProps): JSX.Element => {
+  return (
+    <div className="auth-logo">
+      <img className="auth-logo__image" src={logoWhite} alt="Nimble" />
+      <p className="auth-logo__label">{label}</p>
```

Comment:
@gutakk Not sure why do you desire to put this here, IMO this is not quite related to the logo but more like a page title or a form title. So how about moving it out of this component?
![Screen Shot 2564-06-18 at 19 05 59](https://user-images.githubusercontent.com/1772999/122558495-579cc480-d068-11eb-97b0-4a834f4bfdbf.png)

Reply:
Because I think all auth page have the same layout that's why I named this component as `AuthLogo` but maybe we can change to `AuthTitle` to make it clearer. What do you think?

Reply:
@gutakk How about `page-heder` or `auth-header`, since the `title` sounds more like the component going to cover only the title? 🤔

Reply:
Sounds great. Fixed in 1323a67c2216c104f7d18a323d73c35767a3bdc3